libr/bin/dwarf.c

```diff
@@ -1134,22 +1273,20 @@ static void dump_r_bin_dwarf_debug_abbrev(FILE *f, RBinDwarfDebugAbbrev *da) {
 	}
 	for (i = 0; i < da->length; i++) {
 		int declstag = da->decls[i].tag;
-		fprintf (f, "Abbreviation Code %"PFMT64d" ", da->decls[i].code);
+		fprintf (f, "   %-4"PFMT64d" ", da->decls[i].code);
```

Comment:
not for this PR, but this `fprintf` thing in this entire file is wrong. It should use `r_cons_printf` and not directly output to `stdout`. Otherwise things like internal grep `~` won't work on dwarf commands like `id`.

Reply:
I can throw refactor the printf here, doesn't seem like much of an issue, unless doing it in another PR is wanted

Reply:
So I've refactored it to use r_cons_printf at debug_info related code.

Reply:
Please remove them in this PR, otherwise can be quickly forgotten.